tests/dummy/app/lib/string.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Wasn't sure how to reuse this file from lib in the dummy app without making it part of the public API so I duplicated it.

Reply:
I think this is fine.